### ISSUE TITLE
Q: What's the purpose of this project?

### ISSUE THREAD

**Issue by dodalovic** [HollowMan6/Customize-IBus#1]
Hi, I'm not sure what to do after I install it and what's the purpose of the extension?

Thanks!

**Comment by HollowMan6** [comment]
Hi, this extension is part of a demo that I have done during GSoC 2021 application at OpenSUSE, you can get some information there: https://github.com/openSUSE/mentoring/issues/158

And to be more precise, here's the introduction of [IBus](https://en.wikipedia.org/wiki/Intelligent_Input_Bus), it's for inputting non-latin characters, such as Chinese and Japanese. 

For the IBus theme customization feature, currently IBus theme follows and is decided by the desktop environment theme, this extension aims to make IBus themes separate from current GNOME-Shell theme so that users can customize it with other available GNOME-Shell themes. 

The remaining features are imported from https://github.com/tuberry/ibus-tweaker so that users can customize IBus with my extension conveniently without needs to install other extensions like ibus-tweaker.

So in conclusion, if you do not use IBus, you won't find this extension useful, but for those who uses IBus for their daily inputting, this extension may be a great help.

Thanks!

**Comment by dodalovic** [comment]
Hey, I often need to use different alphabets, such as German, etc. Would your tool help me somehow? I'm lazy to constantly switch the keyboard layout

**Comment by HollowMan6** [comment]
Well, I guess not. I only know that ibus supports Chinese, Japanese and Vietnamese. I don't think they support Germany. For my tool is based on Ibus and is only an customization for IBus, you can go there for help https://github.com/ibus/ibus
